generate-new-strings-pot.sh

```diff
@@ -52,23 +54,33 @@ if [[ "$CI_PULL_REQUEST" ]]; then
 	COMMITSURL=https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/pulls/${CI_PULL_REQUEST##*/}/commits
 	echo "LocalCI - fetching $FILESURL"
 	GH_FILESURL_CONTENT=$(auth_gh_curl $FILESURL)
+
+	# Disable exit on error, this section uses non-0 exit codes
```

Comment:
Maybe rather `Disable exit on error, since this section checks for non-0 exit codes`?